homeassistant/components/lidarr/sensor.py

```diff
@@ -50,25 +48,23 @@ def get_modified_description(
 class LidarrSensorEntityDescriptionMixIn(Generic[T]):
     """Mixin for required keys."""
 
-    value_fn: Callable[[T, str], str]
+    value_fn: Callable[[T, str], str | int]
 
 
 @dataclass
 class LidarrSensorEntityDescription(
-    SensorEntityDescription, LidarrSensorEntityDescriptionMixIn, Generic[T]
+    SensorEntityDescription, LidarrSensorEntityDescriptionMixIn[T], Generic[T]
 ):
     """Class to describe a Lidarr sensor."""
 
-    attributes_fn: Callable[
-        [T], dict[str, StateType | datetime] | None
-    ] = lambda _: None
+    attributes_fn: Callable[[T], dict[str, str] | None] = lambda _: None
     description_fn: Callable[
-        [LidarrSensorEntityDescription, LidarrRootFolder],
-        tuple[LidarrSensorEntityDescription, str] | None,
-    ] = lambda _, __: None
+        [LidarrSensorEntityDescription[T], LidarrRootFolder],
+        tuple[LidarrSensorEntityDescription[T], str] | None,
+    ] | None = None
```

Comment:
This is a small logic change. Even an empty lambda would  have evaluated to true.
https://github.com/home-assistant/core/blob/1a9bcafbd209e166e43a66fb3609b3c14eb9da36/homeassistant/components/lidarr/sensor.py#L121-L123

@tkdrob If you agree, I can tag this PR for the next beta release.